de.html

```diff
@@ -1309,31 +1309,31 @@
 <panel w=500 h=350>
 <pic src="pics/end0.png" sx=0 sy=0></pic>
 <words x=10 y=10 w=450 h=60>
-	To help you get started with Spaced Repetition <i>today</i>,
-	you need to answer four questions:
+	Um dir zu helfen <i>heute</i> mit Spaced Repetition anzufangen,
```

Comment:
```suggestion
	Um dir zu helfen <i>heute</i> mit Intervall-Lernen anzufangen,
```
Der Begriff scheint halbwegs etabliert, macht Sinn und passt von der Länge her. Ich schlage vor bei der ersten Verwendung ne Fußnote auf das lange "Wiederholungen in regelmäßigen Abständen" einzufügen.

Reply:
"Intervall-Lernen" find ich super!